docs/data/date-pickers/custom-field/PickerWithAutocompleteField.tsx

```diff
@@ -42,7 +45,13 @@ function AutocompleteField(props: AutoCompleteFieldProps) {
       return nonNullAdornments[0];
     }
 
-    return <Stack direction="row">{nonNullAdornments}</Stack>;
+    return (
+      <Stack direction="row">
+        {nonNullAdornments.map((adornment, index) => (
+          <React.Fragment key={index}>{adornment}</React.Fragment>
```

Comment:
What does the fragment accomplish here ?

Reply:
The ability to add a `key` to the iterated item in order to avoid the error of a missing `key`. 🤔